sys-utils/libcmt/include/libcmt/ioctl/io-impl.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright Cartesi and individual authors (see AUTHORS)
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef CMT_IOCTL_IO_IMPL_H
+#define CMT_IOCTL_IO_IMPL_H
+#include <stdint.h>
+#include <stddef.h>
+
+struct cmt_io_driver {
+	struct {
```

Comment:
Are these `-impl.h` files going to be installed? If not make sure `make install` does not install them, or move out of `include/libcmt` directory.
If they are, better add a typedef and also not use unnamed structs, this is not friendly for C binding generators.

Reply:
Yes, the idea is to distribute them (the why is on another comment).
Ok, I'll add the typedefs.